### PR TITLE
feat(user-notification): Add user notification client

### PR DESCRIPTION
**Pull request by thordurhhh**
## What

Add user-notification client

## Why

New client for requests to user-notification service

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/clients/user-notification/src/lib/apiConfiguration.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            ? {
+                issuer: idsConfig.issuer,
+                clientId: idsConfig.clientId,
+                clientSecret: idsConfig.clientSecret,
+                scope: config.scope,
+                mode: 'auto',
+              }
+            : undefined,
+        }),
+        basePath: `${config.basePath}/v1`,
```
Comment (baering) [inline libs/clients/user-notification/src/lib/apiConfiguration.ts:34]: Was the `/v1` basePath postfix intentional?

**Review by baering — APPROVED** [review]
LGTM

File: libs/clients/user-notification/src/lib/apiConfiguration.ts (hunk 1 above)
Comment (thordurhhh) [reply libs/clients/user-notification/src/lib/apiConfiguration.ts:34]: Yes, it was. Perhaps it should be generated by the service as with some other services. But it is not auto generated in this case.

**Review by robertaandersen — APPROVED** [review]
LGTM

**Pull request merged**